backend/uasmartsignage/src/main/java/deti/uas/uasmartsignage/Services/FileService.java

```diff
@@ -98,6 +98,34 @@ public Optional<CustomFile> getFileOrDirectoryById(Long id) {
         return file;
     }
 
+    /***
+     * Retrieves and returns CustomFile with the specified Path from the file repository.
+     * 
+     * @param path The Path of the CustomFile to retrieve.
+     * @return The CustomFile with the specified ID, or null if no such file is found.
+     */
+    public Optional<CustomFile> getFileOrDirectoryByPath(String path){
+        path = path.replaceAll("[\n\r]", "_");
+        logger.info("Retrieving file with path: {}", path);
+        Optional<CustomFile> file = fileRepository.findByPath(path);
+        if (file.isEmpty()) {
+            logger.warn("File with Path {} found", path);
+            return Optional.empty();
```

Comment:
Should this be an `error` or `warn` ?
And change the log to express that it did not find the file